### PR TITLE
Fuse tachyons

### PR DESCRIPTION
**Pull request by andrewMacmurray**
add tachyons sections for:

+ title text
+ how it works
+ team
+ partners

reorganise layouts into separate cast and fuse folders

todo: modals for graduates etc

links #112

### CODE REVIEW

File: source/layouts/fuse/_how_it_works.html.haml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+%section#howitworks.bg-navy.pv4
```
Comment (ivan-unfolds) [inline source/layouts/fuse/_how_it_works.html.haml:1]: This section is missing a header.

Comment (ivan-unfolds) [reply source/layouts/fuse/_how_it_works.html.haml:1]: My bad, wrong section. Thought this was nonprofits but see you haven't done that yet.

File: source/layouts/fuse/_partners.html.haml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+%section.pv4.ph4#partners
+  .tc.center
+    = partial "components/section_header", locals: { title: "Supporters and partners", light: true }
+    .flex.flex-wrap.mw8.center
```
Comment (ivan-unfolds) [inline source/layouts/fuse/_partners.html.haml]: Is this missing a justify-center class?

**Review by ivan-unfolds — CHANGES_REQUESTED** [review]
@andrewMacmurray Looks good. Just one comment! Also will probably need to merge `tachyons-setup` back into it.

File: source/layouts/fuse/_partners.html.haml (hunk 2 above)
Comment (andrewMacmurray) [reply source/layouts/fuse/_partners.html.haml]: well spotted

**Comment by andrewMacmurray** [comment]
@ivanmauricio that should be fixed now, have merged with base branch

**Comment by ivan-unfolds** [comment]
Beautiful 🌸

**Pull request merged**